CONTRIBUTING.md

```diff
@@ -18,6 +18,12 @@ us first through the issue tracker with your idea so that we can help out and
 possibly guide you. Coordinating up front makes it much easier to avoid
 frustration later on.
 
+### Implementations
+
+1. For breaking changes, please loop corresponding CODEOWNERs in to make sure they know and can provide a review before a merge to avoid potential regressions.
```

Comment:
As discussed offline, let's skip `breaking` here. 

```suggestion
1. For existing functionality contributions, please loop corresponding CODEOWNERs in to make sure they know and can provide a review before a merge to avoid potential regressions.
```